src/Core/Components/Badge/FluentBadge.razor.css

```diff
@@ -0,0 +1,92 @@
+/*
+    before-top       above        after-top
+    above-start                   above-end
+
+
+    before       center-center    after
+
+
+    before-bottom    below        after-bottom
+    below-start                   below-end
+
+*/
+
+fluent-badge[positioning='before-top'],
+fluent-badge[positioning='above-start'],
+fluent-counter-badge[positioning='before-top'],
+fluent-counter-badge[positioning='above-start'] {
+  position: absolute;
+  top: calc(1 * attr(offset-y px, 0px));
+  left: calc(1 * attr(offset-x px, 0px));
+  transform: translate(-50%, -50%);
+}
+
+fluent-badge[positioning='above'],
+fluent-counter-badge[positioning='above'] {
+  position: absolute;
+  top: calc(1 * attr(offset-y px, 0px));
+  left: calc(50% + 1 * attr(offset-x px, 0px));
+  transform: translate(-50%, -50%);
+}
+
+fluent-badge[positioning='after-top'],
+fluent-badge[positioning='above-end'],
+fluent-counter-badge[positioning='after-top'],
+fluent-counter-badge[positioning='above-end'] {
+  position: absolute;
```

Comment:
You need to add these 2 selectors to have a **default** position when the `Positioning` parameter is `null`.
```css
fluent-badge:not([positioning]),
fluent-counter-badge:not([positioning]) {
```

Reply:
No, when positioning is not set, the badge needs to rendered as-is, where it is.

Reply:
This makes sense, but in this case, if a `ChildContent` exists, you must use the default Top/Right position (as described in the documentation). At this moment, the `positionning` attribute will be undefined and the position is not correct.

Reply:
Setting the default value is now handled in `OnParameterSet`